tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php

```diff
@@ -1144,6 +1144,179 @@ public function f2() {}
                 ',
                 ['elements' => ['const' => ClassAttributesSeparationFixer::SPACING_NONE, 'method' => ClassAttributesSeparationFixer::SPACING_ONE]],
             ],
+            [
+                '<?php
+                    class A
+                    {
+                        const FOO = 1;
+                        const BAR = 2;
+
+                        /** @var int */
+                        const BAZ = 3;
+
+                        /** @var int */
+                        const NEW = 4;
+                    }
+                ',
+                '<?php
+                    class A
+                    {
+                        const FOO = 1;
+
+                        const BAR = 2;
+
+                        /** @var int */
+                        const BAZ = 3;
+
+                        /** @var int */
+                        const NEW = 4;
+                    }
+                ',
+                ['elements' => ['const' => ClassAttributesSeparationFixer::SPACING_HYBRID]],
+            ],
+            [
+                '<?php
+                    class B
+                    {
+                        public $foo;
+
+                        /** @var string */
+                        public $bar;
+                        public $baz;
+                    }
+                ',
+                '<?php
+                    class B
+                    {
+                        public $foo;
+
+                        /** @var string */
+                        public $bar;
+
+                        public $baz;
+                    }
+                ',
+                ['elements' => ['property' => ClassAttributesSeparationFixer::SPACING_HYBRID]],
+            ],
+            [
+                '<?php
+                    class C
+                    {
+                        public function f1() {}
+                        public function f2() {}
+                        public function f3() {}
+
+                        /** @return string */
+                        public function f4() {}
+                    }
+                ',
+                '<?php
+                    class C
+                    {
+                        public function f1() {}
+
+                        public function f2() {}
+
+                        public function f3() {}
+
+                        /** @return string */
+                        public function f4() {}
+                    }
+                ',
+                ['elements' => ['method' => ClassAttributesSeparationFixer::SPACING_HYBRID]],
+            ],
+            [
+                '<?php
+                class Entity
+                {
+                    /**
+                     * @ORM\Column(name="one", type="text")
+                     */
+                    private string $one;
+
+                    /**
+                     * @ORM\Column(name="two", type="text")
+                     */
+                    private string $two;
```

Comment:
Those tests are failing on PHP ≤ 7.3 because they use typed properties. They should be moved to `provideFix74Cases()`.